lib/Module/KInstruction.cpp

```diff
@@ -20,15 +19,8 @@ KInstruction::~KInstruction() {
   delete[] operands;
 }
 
-void KInstruction::printFileLine(llvm::raw_ostream &debugFile) const {
-  if (info->file != "")
-    debugFile << info->file << ":" << info->line;
-  else debugFile << "[no debug info]";
-}
-
-std::string KInstruction::printFileLine() const {
-  std::string str;
-  llvm::raw_string_ostream oss(str);
-  printFileLine(oss);
-  return oss.str();
+std::string KInstruction::getSourceLocation() const {
+  if  (info->file != "")
```

Comment:
Nowadays `.empty()` is preferred over `== ""` and is also more flexible in case we switch to wstring.